host_core/lib/host_core/control_interface/acl.ex

```diff
@@ -0,0 +1,38 @@
+defmodule HostCore.ControlInterface.ACL do
```

Comment:
Does ACL stand for access control list here?

Reply:
Anti-Corruption Layer